auto-check-fix-commit/README.md

```diff
@@ -17,12 +17,19 @@ Reference - https://git-scm.com/docs/git-bisect/
 
 ### Usage
 
-1. Copy scripts `git-bisect-runner.sh` and `git-bisect-script.sh` to the directory of the project containing the fixed flaky test.
+1. Fork/clone this repo with the auto-check-fix-commit module containing the `git-bisect-runner.sh` and `git-bisect-script.sh` scripts.
 
+2. Within `git-bisect-runner.sh`, modify the `git bisect run ./git-bisect-script.sh $test_module $test_case` line to specify your path to `git-bisect-script.sh`, adding in `nohup` command at the beginning of the command to keep process running even after exiting shell/terminal.
```

Comment:
It shouldn't be necessary to modify the path in `git-bisect-runner.sh` -- if both that script and `git-bisect-script.sh` are in the same directory, then `git-bisect-runner.sh` should find its own directory and run `git-bisect-script.sh` from there. (The names for the scripts are not very good, but let's keep them as they are for now.)

Also, it shouldn't be necessary to add another `nohup` to this script if the first one is already run with `nohup`.

Reply:
Got it! I will remove this step.